src/ui/SessionExpiryTracker/SessionExpiryTracker.tsx

```diff
@@ -3,21 +3,21 @@ import React, { useCallback, useEffect, useState } from 'react'
 import { Redirect } from 'react-router-dom'
 
 import config, {
-  COOKIE_SESSION_ID,
+  COOKIE_SESSION_EXPIRY,
   LOCAL_STORAGE_SESSION_EXPIRED_KEY,
   LOCAL_STORAGE_SESSION_TRACKING_KEY,
 } from 'config'
 
-const ONE_MINUTE_MILLIS = 60 * 1000
-const TWO_MINUTES_MILLIS = 2 * ONE_MINUTE_MILLIS
+const ONE_MINUTE_MILLIS = 6 * 1000
```

Comment:
mistake here - left from testing I'd imagine

Reply:
As in the title, still a draft that I was hoping we could try and figure out the banner thing before I cleaned up :)

Reply:
Ahhh gotcha.